src/socialMediaActivity.js

```diff
@@ -30,15 +31,15 @@ export function enablePrivateWindows() {
     privateWindows = true;
 }
 
-/** Unregister old handlers for an event, and register a new one, if necessary.
+/**
+ * Unregister old handlers for an event, and register a new one, if necessary.
  * Unregistering is only necessary when there's already a nonblocking handler registered
  * and we want to convert it to a blocking handler.
  * @param platform - which social media platform the event is for
  * @param eventType - which type of event we're registering
```

Comment:
Perhaps we should use the term "activity type" rather than "event type", since that's consistent with the name of the module and events are a specific concept in WebExtensions.